azerothcore/azerothcore-wotlk
# fix(Core/Arenas): Fixed getting correct battleground status in case o…

…f people leaving arena.

Fixed #7278



## Changes Proposed:
-  Fixed getting correct battleground status in case of people leaving arena

## Issues Addressed:

- Closes #7278

## Tests Performed:

- Not tested

## How to Test the Changes:

1. enter arena 2x2 or 3x3
2. wait for your friend(s) to die or tell him/them to leave
3. (while out of combat) leave arena
4. check your personal score/rating and your arena team score/rating


## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
